crates/red_knot_python_semantic/resources/mdtest/call/callable_instance.md

```diff
@@ -52,7 +52,7 @@ class NonCallable:
     __call__ = 1
 
 a = NonCallable()
-# error: "Object of type `NonCallable` is not callable"
+# error: "Object of type `Unknown | Literal[1]` is not callable (due to union element `Literal[1]`)"
```

Comment:
Another instance of the callable problem. Notice how the usefulness of the error message degrades.